README.md

```diff
@@ -4,6 +4,7 @@ This repository is a [Jekyll](http://jekyllrb.com/) website hosted on [Github Pa
 
 The repository/website is meant to be a point around which users can collaborate on the ROS 2.0 design efforts as well as capture those discussions for posterity.
 
+The best mailing list for discussing these topics is [ros-sig-ng-ros@googlegroups.com](mailto:ros-sig-ng-ros@googlegroups.com) You can view the archives [here](https://groups.google.com/forum/?fromgroups#!forum/ros-sig-ng-ros)
```

Comment:
I think there should be punctuation in this, and it should be on two lines, since it is structured as two sentences:

```
The best mailing list for discussing these topics is [ros-sig-ng-ros@googlegroups.com](mailto:ros-sig-ng-ros@googlegroups.com).
You can view the mailing list archives [here](https://groups.google.com/forum/?fromgroups#!forum/ros-sig-ng-ros).
```

Same in the other file.